### DISCUSSION TITLE
For Xavier bundled initramfs, is there a way to get the log output?

### DISCUSSION THREAD

**Discussion by danielwangksu** [OE4T/meta-tegra#566] [Q&A] [answered]
Hi there,

I have a question, for Xavier we have the init.sh script running after bundled initramfs bootup (https://github.com/OE4T/meta-tegra/blob/master/recipes-core/initrdscripts/tegra-minimal-init/init-boot.sh). However, I can only find the script output from serial console, after the system fully bootup there is no log saved. Is there a way to save the script's output? Thanks

**Comment by madisongh** [comment, marked as answer]
You'd have to put something together yourself for that, or replace the minimal script with a more robust initramfs framework that supports logging.  If you roll your own, you could use I/O redirects to log to a file (and make sure that file survives the switch_root) or the kernel log.

**Reply by danielwangksu** [reply]
Thank you @madisongh. I think initramfs framework is a good idea. I tried I/O redirection but it is not ideal.
